runners/direct-java/src/main/java/org/apache/beam/runners/direct/ParDoEvaluatorFactory.java

```diff
@@ -87,7 +87,7 @@ public void cleanup() throws Exception {
               stepContext,
               application,
               application.getInput().getWindowingStrategy(),
-              fnManager.get(),
+              fnManager.get().getFn(),
```

Comment:
This particular change should go in the other commit, and rebase this commit to happen before it.